code/modules/mob/mob_defines.dm

```diff
@@ -127,7 +127,7 @@
 
 	mouse_drag_pointer = MOUSE_ACTIVE_POINTER
 
-	var/update_icon = 1 //Set to 1 to trigger update_icon() at the next life() call
+	//var/update_icon = 1 //Set to 1 to trigger update_icon() at the next life() call
```

Comment:
if it's deprecated this should be removed fully, i figure the commenting-out was just to check if anywhere used it?